release/release_tests.yaml

```diff
@@ -1044,7 +1044,7 @@
 
   run:
     timeout: 1000
-    script: chmod +x ./run_llama_ft.sh && ./run_llama_ft.sh --size=7b --as-test
+    script: chmod +x ./run_llama_ft.sh && ./run_llama_ft.sh --size=7b --as-test && ./run_llama_ft.sh --size=7b --lora --as-test
```

Comment:
We should create a separate lora release tests item not concatenate the existing one with the lora variation.

Reply:
Upon failure you won't know which one failed (lora or full) and you have to do log reading which is painful.  Please fix this.

Reply:
The idea was to save CI cost, but I'll do as you say.